### PR TITLE
Update membership

### PR DESCRIPTION
**Pull request by scsven**
Signed-off-by: Xiangyu Wang <wxyucs@gmail.com>

### CODE REVIEW

File: community/site/en/communityArticles/contributor_group/membership.md [hunk 1]
```diff
@@ -63,5 +66,20 @@ Maintainers are able to both review and approve code contributions. While code r
...
 
+-   Reviewer of the codebase for at least 3 months
+-   Primary reviewer for at least 10 PRs to the codebase
 -   Main contributor to the codebase
--   Actively participate in code review
+-   Nominated by a maintainer or toc members owner 
+
+
+
+## TOC(technical oversight committee) members
```
Comment (xiaofan-luan) [inline community/site/en/communityArticles/contributor_group/membership.md:76]: Lets' change TOC to TSC(Technical Steering Committee )

File: community/site/en/communityArticles/contributor_group/membership.md [hunk 2]
```diff
@@ -63,5 +66,20 @@ Maintainers are able to both review and approve code contributions. While code r
...
+
+The TOC is elected by the community. The TOC’s work includes: 
+
+-   Ensure continued health of Milvus project
+-   Set technical direction and priorities for the community
+-   Mentor and guide maintainers, reviewers, and contributors to the subproject
```
Comment (xiaofan-luan) [inline community/site/en/communityArticles/contributor_group/membership.md:82]: Mentor and guide maintainers, reviewers, and contributors to the  project

**Review by xiaofan-luan — CHANGES_REQUESTED** [review]

**Comment by scsven** [comment]
@xiaofan-luan Updated. Please review again.

File: community/site/en/communityArticles/contributor_group/membership.md [hunk 3]
```diff
@@ -9,9 +9,9 @@ This doc outlines the various responsibilities of contributor roles in The Milvu
...
 | ------------- | --------------------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
 | Contributor   | Contributor in the community            | Made more than one contributions to the Milvus project       | List in [All contributors](https://github.com/milvus-io/milvus#all-contributors) |
-| Active Contributor | Active contributor in the community     | Invited by reviewer and multiple contributions to the Milvus project | milvus-io GitHub org member                                  |
 | Reviewer      | Review contributions from other members | History of review and authorship in a subproject             | [OWNERS] file reviewer entry                                 |
 | Maintainer    | Contributions acceptance approval       | Highly experienced active reviewer and contributor to a subproject | [OWNERS] file approver entry                                 |
+| TOC members | Ensure continued health of The Milvus Project | Demonstrated responsibility and excellent technical judgement for The Milvus Project |  |
```
Comment (xiaofan-luan) [inline community/site/en/communityArticles/contributor_group/membership.md:14]: change it to TSC

File: community/site/en/communityArticles/contributor_group/membership.md [hunk 4]
```diff
@@ -50,8 +36,10 @@ Reviewers are able to review code for quality and correctness on some part of a
 
 **Requirements**
 
--   Have made over 30 commits to the codebase
--   Actively participate in code review
+-   Active contributor for at least 3 months
+-   Review for at least 5 PRs
+-   Have made over 20 commits to the codebase
+-   Nominate by maintainers or TOC members
```
Comment (xiaofan-luan) [inline community/site/en/communityArticles/contributor_group/membership.md:42]: change to TSC

File: community/site/en/communityArticles/contributor_group/membership.md [hunk 5]
```diff
@@ -63,5 +51,20 @@ Maintainers are able to both review and approve code contributions. While code r
 
 **Requirements**
 
+-   Reviewer of the codebase for at least 3 months
+-   Primary reviewer for at least 10 PRs to the codebase
 -   Main contributor to the codebase
--   Actively participate in code review
+-   Nominated by a maintainer or toc members owner
```
Comment (xiaofan-luan) [inline community/site/en/communityArticles/contributor_group/membership.md:57]: change it to TSC

File: community/site/zh-CN/communityArticles/contributor_group/membership.md [hunk 6]
```diff
@@ -6,12 +6,12 @@ id: membership.md
...
-| Reviewer           | 审查其他成员的 PR | 在任一仓库多次 Review 和贡献代码                | [OWNERS] 文件中 reviewers 字段                               |
-| Maintainer         | 批准其他成员的 PR | 经验丰富的 Reviewer，非常清楚地了解某一部分代码 | [OWNERS] 文件中 approvers 字段                               |
+| 角色               | 职责                   | 要求                                            | 定义                                                         |
+| ------------------ | ---------------------- | ----------------------------------------------- | ------------------------------------------------------------ |
+| Contributor        | 社区的贡献者           | 在 Milvus 项目或子项目做出过一次以上的贡献      | 在 milvus 仓库的 [All contributors 列表](https://github.com/milvus-io/milvus#all-contributors) 中列出 |
+| Reviewer           | 审查其他成员的 PR      | 在任一仓库多次 Review 和贡献代码                | [OWNERS] 文件中 reviewers 字段                               |
+| Maintainer         | 批准其他成员的 PR      | 经验丰富的 Reviewer，非常清楚地了解某一部分代码 | [OWNERS] 文件中 approvers 字段                               |
+| TOC members        | 确保Milvus项目健康发展 | 对 Milvus 项目表现出责任感和出色的技术视野      |                                                              |
```
Comment (xiaofan-luan) [inline community/site/zh-CN/communityArticles/contributor_group/membership.md:14]: change to TSC

File: community/site/zh-CN/communityArticles/contributor_group/membership.md [hunk 7]
```diff
@@ -61,5 +48,19 @@ Maintainer 负责审查和批准代码提交，主要考虑代码的依赖关系
 
 **要求**
 
--   这个目录的主要贡献者
--   积极参与这个目录相关的 Code Review
\ No newline at end of file
+-   成为 Reviewer 3 个月以上的时间
+-   作为主要 Reviewer 参与 10 个以上 PR 的审查
+-   这个目录代码的主要贡献者
+-   由 Maintainer 或者 TOC 成员提名
```
Comment (xiaofan-luan) [inline community/site/zh-CN/communityArticles/contributor_group/membership.md:54]: TSC

File: community/site/zh-CN/communityArticles/contributor_group/membership.md [hunk 8]
```diff
@@ -48,8 +33,10 @@ Reviewer 负责审查代码质量和正确性，他们需要对负责的代码
 
 **要求**
 
--   在这个目录下有 30 个以上的 commit 贡献
--   积极参与这个目录相关的 Code Review
+-   成为 Active Contributor 3 个月以上的时间
+-   参与 5 个以上 PR 的审查
+-   在这个目录下有 20 个以上的 commit 贡献
+-   由 Maintainer 或者 TOC 成员提名
```
Comment (xiaofan-luan) [inline community/site/zh-CN/communityArticles/contributor_group/membership.md:39]: TSC

**Review by xiaofan-luan — CHANGES_REQUESTED** [review]

**Pull request merged**